com.spotify.Client.json

```diff
@@ -150,6 +150,50 @@
                 }
             ]
         },
+        /* THIS CANNOT BE UPDATED UNTIL SPOTIFY SUPPORTS ffmpeg 5 ------------------------------------------------------- */
+        {
+            "name": "ffmpeg-4.4",
+            "config-opts": [
+                "--libdir=\"/app/lib/\"",
```

Comment:
I'm pretty sure this line isn't needed as well. Other than that it looks fine, thx for the patience!

Reply:
I will double check.

Reply:
nope it doesnt look like it. I think this was left over from when i was trying to get it to compile. Ill double check and ill update

Reply:
we are good, ill push the commit now